integration-testing/rnode_testing/profiling.py

```diff
@@ -2,8 +2,9 @@
 from functools import wraps
 import logging
 from rnode_testing.util import log_box
+from typing import Dict, List
 
-PROF_DATA = {}
+PROF_DATA:Dict[str, List] = {}
```

Comment:
```suggestion
PROF_DATA: Dict[str, List] = {}
```
Space after `:` is an established convention in the Python world